packages/playground/php-cors-proxy/cors-proxy-functions.php

```diff
@@ -411,6 +411,8 @@ function should_respond_with_cors_headers($host, $origin) {
         'https://playground.wordpress.net',
         'http://localhost',
         'http://127.0.0.1',
+        'http://127.0.0.1:5400',
```

Comment:
Turns out the local proxy wasn't proxying the network calls – the origin contains a port number

Reply:
Maybe we could also clean up the port-less entries above

Reply:
Actually no, I'll leave them for testing on port 80.